core/sail/fts/lucene-api/src/main/java/org/eclipse/rdf4j/sail/lucene/QuerySpec.java

```diff
@@ -54,6 +62,40 @@ public QuerySpec(StatementPattern matchesPattern, StatementPattern queryPattern,
 		this.subject = subject;
 		this.queryString = queryString;
 		this.propertyURI = propertyURI;
+		if (matchesPattern != null)
+			this.matchesVarName = matchesPattern.getSubjectVar().getName();
```

Comment:
This is really minor, but we have an implicit agreement to _always_ use curly braces for if-statements, even if the body is only a single line.